wagtailio/static/sass/components/_modal.scss

```diff
@@ -0,0 +1,154 @@
+.modal {
+    $root: &;
+
+    &--get-started {
+        // hide the modal if it's open below medium breakpoint
```

Comment:
What happens then?

Reply:
The mobile menu functionality will take precedence